.github/workflows/release.yaml

```diff
@@ -5,14 +5,18 @@ on:
 
 env:
   REGISTRY: ghcr.io
-  IMAGE_NAME: ${{ github.repository }}
 
 jobs:
   build-docker:
     runs-on: ubuntu-latest
     permissions:
       contents: read
       packages: write
+    strategy:
+      matrix:
+        include:
+        - image: ffprobe-slim
+          buildContext: ./ffprobe-slim
```

Comment:
Could probably optimize it so that it only builds when anything inside the context changes.
Makes no difference for now but will matter when we add more variations.